chrome/global/colors.css

```diff
@@ -129,6 +129,11 @@
   --urlbar-box-focus-bgcolor: inherit !important;
 }
 
+/* Change Colour of page when loading */
+:root:not(:-moz-lwtheme) #tabbrowser-tabpanels {
```

Comment:
I think it would be better to move this to the `browser.css` file, as it is only needed when the rounded corners tweak is enabled.

Also remove the `:root:not(:-moz-lwtheme)` part so it applies even if a Firefox colour theme is enabled.

Reply:
Done.